### PR TITLE
fix: auto remove 7 days ago log files

### LINKED ISSUE #23: egg-alinode: auto remove old log files
alinode won't delete the log files, egg-alinode plugin should delete 7 days ago log files.

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines
##### Description of change

closes https://github.com/eggjs/egg/issues/23

---

This change is [Reviewable](https://reviewable.io/reviews/eggjs/egg-alinode/2)

### CODE REVIEW

File: app/schedule/removeLogs.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  const logger = app.coreLogger;
+
+  exports.schedule = {
+    type: 'worker', // only one worker run this task
+    cron: '0 0 * * *', // run every day at 00:00
+  };
+  exports.task = function* () {
+    const logdir = app.config.alinode.logdir;
+    const maxDays = 7;
```
Comment (fengmk2) [inline app/schedule/removeLogs.js:18]: 只保留7天

File: app/schedule/removeLogs.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const path = require('path');
+const fs = require('mz/fs');
```
Comment (fengmk2) [inline app/schedule/removeLogs.js:4]: mz 依赖没加。。。

**Comment by fengmk2** [comment]
1.0.2

**Comment by atian25** [comment]
这个当初怎么不直接用 egg-logrotator 配置？

**Pull request merged**